web/src/main/java/se/inera/intyg/webcert/web/service/facade/question/impl/ArendeToQuestionFacadeServiceImpl.java

```diff
@@ -50,6 +51,7 @@
 import se.inera.intyg.webcert.web.service.facade.question.util.QuestionConverter;
 
 @Service(value = "ArendeToQuestionFacadeService")
+@Transactional
```

Comment:
Im curious to why the transactional annotation had to be added for this service?

Reply:
I got an error otherwise. 

![MicrosoftTeams-image (6)](https://github.com/sklintyg/webcert/assets/54178725/5a03d271-42a6-4f97-bae4-2181608b966a)

Reply:
I think you could change kontaktinfo to EAGER. Same as for komplettering. Then we don't need to add the @Transactional annotation.

Reply:
If I add the same as for komplettering I get this: Unable to build Hibernate SessionFactory; nested exception is org.hibernate.loader.MultipleBagFetchException: cannot simultaneously fetch multiple bags: [se.inera.intyg.webcert.persistence.arende.model.Arende.komplettering, se.inera.intyg.webcert.persistence.arende.model.Arende.kontaktInfo]

Reply:
It seems to be a behaviour of Hibernate... They way it handles EAGER. There are solutions/work-arounds and if you want to avoid the @Transactional I would suggest you also add this:

`@Fetch(value = FetchMode.SUBSELECT)`

So it looks something like this in the end:

![image](https://github.com/sklintyg/webcert/assets/46641204/b55ad711-5c87-4323-9c2e-b6776724bd48)

References:
https://hibernate.atlassian.net/browse/HHH-1718
https://stackoverflow.com/questions/4334970/hibernate-throws-multiplebagfetchexception-cannot-simultaneously-fetch-multipl